tests/test_history.py

```diff
@@ -1,96 +1,103 @@
 """Tests the xonsh history."""
+# pylint: disable=protected-access
 from __future__ import unicode_literals, print_function
 import io
 import os
 import sys
 
 import nose
-from nose.tools import assert_equal, assert_true
+from nose import tools as nose_tools
```

Comment:
Also, I see I may not have answered @scopatz's question:

> What is the reason for renaming the module?

The reason is to avoid namespace conflicts. In this case there is a xonsh.tools module and in general you rename the external module on import rather than an import from within the project.